src/main/java/usersandpostsCRUD/demo/exception/PostNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package usersandpostsCRUD.demo.exception;
+
+public class PostNotFoundException extends RuntimeException {
+    public PostNotFoundException(Long id) {
+        super("Post with id " + id + " not found");
```

Comment:
Have you heard about String.format() function?
Create the exception message using string formst

Reply:
Ohh.. not one of the previous comment is resolved. 

EXPLANATION
In java the String wrapper class gives you out od the box a lot of functions like the one I am suggesting you to use -> String.format()

In the super() method, instead of using a + to combine multiple strings you can use the String.format() function.

How to use the function
String.format("Post with id %s is not found", id), where %s is the wildcard to replace the id dynamically.

In a case where you need to include multiple parameters you will do it like the followinf example

Strin.format("one %s, two %s, three %s", ONE, TWO, THREE)

%s -> is a wildcard for a String
%d -> is a wildcard for a Long, Double, Integer


Choose what you need